godtools/Assets.xcassets/Features/User Badge Icons/articles-1.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "filename" : "Group 793.pdf",
```

Comment:
hmm, the pdf format for SVGs was mentioned in one kotlin multiplatform shared image library I had found. Maybe I will take a look at it later to try it out.

Reply:
How do PDF files work in Xcode?  Do they still get bundled with the app?  I would like to look into this one a little more, my hope is they don't increase the app size more than using compressed PNG files.

Reply:
I'm a little hesitant about PDF files in Xcode.  Here are some pitfalls to look out for (https://medium.com/livefront/pdf-pitfalls-in-xcode-dc372bc0df5).

Also, it looks like XCode 12 may support SVG which would be worth looking into more https://www.avanderlee.com/xcode/svg-image-assets/

Reply:
the SVG support is available starting with iOS 13 according to the second link you posted, does that line up with our iOS minimum version for GodTools?

Reply:
Yep GodTools is supporting minimum of iOS 14.